Sources/SQLite/Database/SQLiteStatement.swift

```diff
@@ -52,21 +52,20 @@ internal struct SQLiteStatement {
                 }
             }
         }
-
     }
-
+    
     internal func getColumns() throws -> [SQLiteColumn]? {
         var columns: [SQLiteColumn] = []
-
+        
```

Comment:
could you enable blank line trimming in xcode? that way we can remove all the noise from these diffs.